src/v2/guide/migration.md

```diff
@@ -1,53 +1,58 @@
 ---
-title: Migration from Vue 1.x
+title: Migrasi dari Vue 1.x
 type: guide
 order: 701
 ---
 
 ## FAQ
 
-> Woah - this is a super long page! Does that mean 2.0 is completely different, I'll have to learn the basics all over again, and migrating will be practically impossible?
+> Woah - halaman ini sangat panjang! Apakah ini berarti 2.0 benar-benar beda, Saya harus mempelajari semua dasarnya dari awal lagi, dan migrasi tidak mungkin dilakukan?
 
-I'm glad you asked! The answer is no. About 90% of the API is the same and the core concepts haven't changed. It's long because we like to offer very detailed explanations and include a lot of examples. Rest assured, __this is not something you have to read from top to bottom!__
+Saya senang Anda bertanya! Jawabannya adalah tidak. Sekitar 90% API masih sama dan konsep inti tidaklah berubah. Halaman ini panjang karena kami ingin memberikan penjelasan yang sangat mendetil dan berisikan banyak contoh-contoh. Tenanglah, __ini bukan sesuatu yang Anda harus baca dari atas kebawah!__
 
-> Where should I start in a migration?
+> Darimana saya harus memulai sebuah migrasi?
 
-1. Start by running the [migration helper](https://github.com/vuejs/vue-migration-helper) on a current project. We've carefully minified and compressed a senior Vue dev into a simple command line interface. Whenever they recognize an obsolete feature, they'll let you know, offer suggestions, and provide links to more info.
+1. Awali dengan menjalankan [pembantu migrasi](https://github.com/vuejs/vue-migration-helper) dalam project saat ini. Kami sudah sangat berhati-hati untuk mengecilkan dan mengkompres seorang senior Vue developer menjadi sebuah *command line interface*. Kapan pun mereka mendeteksi adanya fitur yang usang, mereka akan memberitahu Anda, memberikan nasihat, dan menyediakan *links* ke lebih banyak informasi.
```

Comment:
link = tautan

Saya tidak yakin dengan "a senior Vue dev" jika diartikan "seorang senior Vue developer". Ada masukan @nusendra ?

Reply:
waiting for this ya

Reply:
Ya, saya gak yakin dengan kalimatnya

Reply:
Okey @isatrio 

Sebenarnya saya yakin itu adalah "seorang seior Vue developer"
Karena saat saya perhatikan dihalaman ini memang banyak humor-nya